packages/web-config-server/src/apiV1/dataBuilders/modules/covid-samoa/valueAndPercentageByDataValueByFlightDate.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+import moment from 'moment';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import {
+  Flight,
+  getTotalNumPassengers,
+  getPassengersPerDataValue,
+  FLIGHT_DATE,
+} from './flight';
+
+export class ValueAndPercentageByDataValueByFlightDate extends DataBuilder {
+  async build() {  
+    const dataElementCodes = this.getDataElementCodes();
+
+    const events = await this.fetchEvents({ useDeprecatedApi: false, dataElementCodes });
```

Comment:
Gosh we should really properly deprecate this api so we don't have to keep specifying otherwise huh? 😝